### PR TITLE
Improve the README and some site docs

### PR DESCRIPTION
**Pull request by samozzy**
Update with latest information and improve layout

### CODE REVIEW

File: README.md
Comment (BenWoodford76) [inline README.md:30]: Do we still do deploy previews?

**Review by BenWoodford76 — APPROVED** [review]

**Comment by BenWoodford76** [comment]
i'll let you merge this if you want @samozzy

File: README.md
Comment (samozzy) [reply README.md:30]: I would quite like them to come back 😛

**Pull request merged**
